btree/btree.c

```diff
@@ -5358,17 +5883,79 @@ static void vkvv_inode_del(const struct nd *node, int idx)
 	}
 }
 
-static void vkvv_del(const struct nd *node, int idx)
+static void vkvv_del(const struct nd *node, int idx, struct m0_be_tx *tx)
 {
 	struct vkvv_head *h = vkvv_data(node);
 
-	if (vkvv_level(node) == 0)
-		vkvv_lnode_del(node,idx);
+	if (vkvv_addrtype_get(node) == INDIRECT_ADDRESSING)
+		vkvv_indirect_rec_del(node, idx, tx);
+	else if (vkvv_level(node) == 0)
+		vkvv_lnode_del(node, idx);
 	else
-		vkvv_inode_del(node,idx);
+		vkvv_inode_del(node, idx);
 	h->vkvv_used--;
 }
 
+static void vkvv_indirect_move(struct nd *src, struct nd *tgt,
```

Comment:
Move this function contents in generic_indir_addr_move()

Reply:
Please review the comment in bnode_indirect_move().